Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/Inputs/SEC101_006.GitHubPat.ps1

```diff
@@ -14,3 +14,4 @@ var myGitHubPersonalAccessToken = "ff34885a8624460a855540c6592698d2f1812843";
 # Link style
 # This should trigger, but it should report as NoMatch
 https://github.com/microsoft/sarif-pattern-matcher/commit/4196919fb12cc4aa8ec0159a746c1e03b4ffa89c
+https://avatars0.githubusercontent.com/u/16199012?u=4196919fb12cc4aa8ec0159a746c1e03b4ffa89c&v=4
```

Comment:
>githubusercontent [](start = 17, length = 17)

we might want to look at preemptively protecting against github links browsing against specific commit history?

[https://github.com/microsoft/sarif-pattern-matcher/tree/7af498925c89a9e8c07fb8a65223e89d513d4eea](https://github.com/microsoft/sarif-pattern-matcher/tree/7af498925c89a9e8c07fb8a65223e89d513d4eea)